great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -315,9 +318,14 @@ def __init__(
                 ge_cloud_access_token=ge_cloud_access_token,
                 ge_cloud_organization_id=ge_cloud_organization_id,
             )
+        elif self._context_root_directory and project_config:
+            self._data_context = BridgeFileDataContext(  # type: ignore[assignment]
+                project_config=project_config,
+                context_root_dir=context_root_dir,  # type: ignore[arg-type]
+                runtime_environment=runtime_environment,
+            )
```

Comment:
`BridgeFileDataContext` allows for both `project_config` and `context_root_dir` to be defined.